data/h3_raster_importer/tiff_folder_to_h3_table.py

```diff
@@ -0,0 +1,145 @@
+    'float64': 'double precision'
+}
+BLOCKSIZE=512
+
+logging.basicConfig(level=logging.INFO)
+
+def slugify(s):
+    return re.sub("[^0-9a-z_]+", "_", s.lower())
+
+
+def gen_raster_h3(raster_list, h3_res, geo=False):
+    """Convert a list of identically formatted rasters to H3
+    
+    A function for efficiently turning a set of rasters into an H3 table.
+    
+    Takes a list of 1-band rasters with identical projection/transform.
+    Reads each raster in blocks, and converts to h3 (nearest to centroid).
+    Yields a dataframe with an h3index and one column for each raster's value.
+
+    Args:
+        raster_list: list of paths to rasters
+        h3_res: h3 resolution to use for resampling
+    
+    Yields:
+        A Pandas dataframe for each raster block (usu. 512x512) with an 
+        h3index and one column for each raster's value.
+    """
+    readers = [rio.open(r) for r in raster_list]
+    names = [slugify(os.path.splitext(os.path.basename(r))[0]) for r in raster_list]
+    
+    base = readers[0]
+    for j in range(ceil(base.height/BLOCKSIZE)):
+        for i in range(ceil(base.width/BLOCKSIZE)):
+            window = rio.windows.Window(i*BLOCKSIZE, j*BLOCKSIZE, BLOCKSIZE, BLOCKSIZE)
+            w_transform = rio.windows.transform(window, base.transform)
+            dfs = []
+            for src in readers:
+                if src.transform != base.transform:
+                    raise ValueError("Transforms do not match")
+                arr = src.read(1, window=window)
+                _df = h3ronpy.raster.raster_to_dataframe(arr, w_transform, h3_res, nodata_value=src.profile['nodata'], compacted=False, geo=geo)
+                dfs.append(_df.set_index('h3index')['value'])
+            df = pd.concat(dfs, axis=1)
+            logging.info(f'Reading block {j}, {i}: h3index count {len(df)}')
+            if len(df):
+                df.columns = names
+                df.index = pd.Series(df.index).apply(lambda x: hex(x)[2:])
+                # cast h3index from int64 to hex string
+                yield df
+    for src in readers:
+        src.close()
+
+
+def load_raster_list_to_h3_table(raster_list, table, h3_res):
+    conn = psycopg2.connect(
+        host=os.getenv('API_POSTGRES_HOST'),
+        port=os.getenv('API_POSTGRES_PORT'),
+        user=os.getenv('API_POSTGRES_USERNAME'),
+        password=os.getenv('API_POSTGRES_PASSWORD')
+    )
+    cursor = conn.cursor()
+    cursor.execute(f"DROP TABLE IF EXISTS {table};")
+    
+    first = True
+
+    for block_df in gen_raster_h3(raster_list, h3_res):
+        if first:
+            cols = zip(block_df.columns, block_df.dtypes)
+            schema = ', '.join([f"{col} {DTYPES_TO_PG[str(dtype)]}" for col, dtype in cols])            
+            cursor.execute(f"CREATE TABLE {table} (h3index h3index PRIMARY KEY, {schema});")
+            conn.commit()
+            logging.info(f"Created table {table} with columns {', '.join(block_df.columns)}")
```

Comment:
I'd say you are generating the table and the columns depending on downloaded data, but both table and column names are relevant from the API point of view in order to access this data. Could we define some naming or make it match [the data model](https://dbdiagram.io/d/60bf8dd0b29a09603d1883ef)?